tests/templates/icmp_responder.conf.j2

```diff
@@ -1,6 +1,7 @@
 [program:icmp_responder]
-command=/usr/bin/python /opt/icmp_responder.py {{ icmp_responder_args }}
+command=source /root/env-python3/bin/activate && python /opt/icmp_responder.py {{ icmp_responder_args }}
 process_name=icmp_responder
```

Comment:
Is it possible to run `/root/env-python3/bin/python /opt/icmp_responder.py` instead?
I think it doesn't need to do source and set environment here.

Reply:
> Is it possible to run `/root/env-python3/bin/python /opt/icmp_responder.py` instead? I think it doesn't need to do source and set environment here.

Both will do the trick here, is sourcing the `activate` more safe here as it will build up the full simulated environment like `PATH`?